region/crateria/east/Bowling Alley Path.json

```diff
@@ -437,6 +598,148 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Leave With Runway - Frozen Choot",
+      "requires": [
+        "h_canFrozenEnemyRunway",
+        "Gravity"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 16,
+          "openEnd": 1,
+          "steepUpTiles": 4,
+          "steepDownTiles": 2
+        }
+      },
+      "note": "Freeze the Choot near the end of its descent or immediately after it jumps in order to be able to run onto and off of it without a problem."
+    },
+    {
+      "link": [2, 2],
+      "name": "Leave With Runway, Suitless - Frozen Choot",
+      "requires": [
+        "h_canFrozenEnemyRunway"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 11,
+          "openEnd": 1,
+          "steepUpTiles": 2,
+          "steepDownTiles": 1
+        }
+      },
+      "note": "Freeze the Choot near the end of its descent or immediately after it jumps in order to be able to run onto and off of it without a problem.",
+      "devNote": "The submerged tiles don't contribute to the runway length."
+    },
+    {
+      "link": [2, 2],
+      "name": "Bowling Alley Path Double Frozen Choot Runway (Right)",
+      "notable": true,
+      "requires": [
+        "h_canFrozenEnemyRunway",
+        "Gravity"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 23,
+          "openEnd": 1,
+          "steepUpTiles": 6,
+          "steepDownTiles": 4
+        }
+      },
+      "reusableRoomwideNotable": "Bowling Alley Path Double Frozen Choot Runway",
+      "note": [
+        "Freeze two Choots to significantly increase the length of the runway.",
+        "Freeze each near the end of its descent or immediately after it jumps in order to be able to run onto and off of it without a problem."
+      ]
+    },
+    {
+      "link": [2, 2],
+      "name": "Bowling Alley Path Double Frozen Choot Runway, Suitless (Right)",
+      "notable": true,
+      "requires": [
+        "h_canFrozenEnemyRunway"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 14,
```

Comment:
It feels like a 16 runway where a stutter 3 gets an in room shortcharge. Going in and out of water is a unique thing, so its hard to actually know.
The double freeze is fine for extreme, single freeze is probably ok on expert.  I don't know how to go about a triple freeze without the first ones thawing.

Reply:
I used a crouch+angle down shot while standing between the two of them.
You can also use XRay to buffer the shot but then you need to avoid getting hit from the other choot.